lib/logstash/filters/kv.rb

```diff
@@ -411,8 +411,10 @@ def filter(event)
     return if kv.empty?
```

Comment:
If you fancy, removing this line should satisfy https://github.com/logstash-plugins/logstash-filter-kv/issues/11 too.

Reply:
good point. my concern is with BWC but we could make this a major version bump in that respect and have better consistency in the plugin behaviour.